op-e2e/system/e2esys/setup.go

```diff
@@ -116,6 +116,7 @@ func DefaultSystemConfig(t testing.TB, opts ...SystemConfigOpt) SystemConfig {
 	require.NoError(t, err)
 	deployConfig := config.DeployConfig(sco.AllocType)
 	deployConfig.L1GenesisBlockTimestamp = hexutil.Uint64(time.Now().Unix())
+	deployConfig.L2GenesisBlobTimeOffset = nil
```

Comment:
Then you need to enable it in l2 blob e2e test [here](https://github.com/ethstorage/optimism/blob/9d72b4a2cc0457256957b3b7cede96ee52e6efc2/op-e2e/l2blob/l2blob_test.go#L75).

Reply:
Is it enabled already [here](https://github.com/ethstorage/optimism/blob/9d72b4a2cc0457256957b3b7cede96ee52e6efc2/op-e2e/l2blob/l2blob_test.go#L82)?